firefox-ios/Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -145,15 +146,31 @@ final class SearchSettingsTableViewController: ThemedTableViewController, Featur
 
         switch section {
         case .defaultEngine:
-            engine = model.defaultEngine
-            cell.editingAccessoryType = .disclosureIndicator
-            cell.accessibilityLabel = .Settings.Search.AccessibilityLabels.DefaultSearchEngine
-            cell.accessibilityValue = engine.shortName
-            cell.textLabel?.text = engine.shortName
-            cell.imageView?.image = engine.image.createScaled(IconSize)
-            cell.imageView?.layer.cornerRadius = 4
-            cell.imageView?.layer.masksToBounds = true
-            cell.applyTheme(theme: themeManager.currentTheme)
+            switch indexPath.item {
+            case DefaultSearchItem.defaultEngine.rawValue:
+                engine = model.defaultEngine
+                cell.editingAccessoryType = .disclosureIndicator
+                cell.accessibilityLabel = .Settings.Search.AccessibilityLabels.DefaultSearchEngine
+                cell.accessibilityValue = engine.shortName
+                cell.textLabel?.text = engine.shortName
+                cell.imageView?.image = engine.image.createScaled(IconSize)
+                cell.imageView?.layer.cornerRadius = 4
+                cell.imageView?.layer.masksToBounds = true
+                cell.applyTheme(theme: themeManager.currentTheme)
+            case DefaultSearchItem.defaultSuggestions.rawValue:
+                if !featureFlags.isFeatureEnabled(.firefoxSuggestFeature, checking: .buildAndUser) {
+                    buildSettingWith(
+                        prefKey: PrefsKeys.SearchSettings.showSearchSuggestions,
+                        defaultValue: model.shouldShowSearchSuggestions,
+                        titleText: String.localizedStringWithFormat(
+                            .Settings.Search.ShowSearchSuggestions
+                        ),
+                        cell: cell,
+                        selector: #selector(didToggleSearchSuggestions)
+                    )
+                }
+            default: break
```

Comment:
I believe we want to move this section down with the private toggle and should be enabled for all cases.